containers/hbadeals/install.R

```diff
@@ -1,4 +1,4 @@
 Sys.setenv(TAR = "/bin/tar")
-devtools::install_github("TheJacksonLaboratory/HBA-DEALS", upgrade="never")
+devtools::install_github("TheJacksonLaboratory/HBA-DEALS@5bbd9ca", upgrade="never")
```

Comment:
could you elaborate about this change?

Reply:
> could you elaborate about this change?

Yes for sure. I took the latest commit of the HBA-DEALS repo and making a reference to this snapshot of your code. This will ensure that we know which version of the HBA-DEALS code we are running.

From the [`?devtools::install_github()` docs](https://www.rdocumentation.org/packages/devtools/versions/1.13.6/topics/install_github), the syntax can be 

> [..] in the format username/repo[/subdir][@ref|#pull]. Alternatively, you can specify subdir and/or ref using the respective parameters (see below); if both are specified, the values in repo take precedence.

```R
devtools::install_github(username/repo@ref)
```

the default commit grabbed by the function, is master and will take the latest commit from master. This relative reference to the code changes, so in order to know exactly it's good to have an immutable reference, like the commit hash.